tests/Estonianpin/Utils/UtilsTest.php

```diff
@@ -272,4 +272,20 @@ public function testGetGenderAndCenturyIdentificationNumberExceptionThrowingOnIn
         $this->utils->getGenderAndCenturyIdentificationNumber(2014, 'hermaphrodite');
     }
 
+    /**
+     * @covers \Lkallas\Estonianpin\Utils\Utils::getPinsGeneratorForRange()
```

Comment:
Unnecessary parenthesis in function name, refer to other tests @covers annotations.